scripts/release.py

```diff
@@ -224,7 +224,7 @@ def next_revision(self):
 
 
 class Release(object):
-    def __init__(self, tag, github_token=None):
+    def __init__(self, tag, github_token=None, next_version=None):
```

Comment:
It would be more consistent to call it `next_release` instead of `next_version`.